lodash.src.js

```diff
@@ -11156,6 +11156,211 @@
+     */
+    function divide(dividend, divisor) {
+      return dividend / divisor;
+    }
+
+    /**
+     * Calculates the remainder of a number divided by another.
+     *
+     * @static
+     * @memberOf _
+     * @category Math
+     * @param {number} dividend The number to divide.
+     * @param {number} divisor The number to divide by.
+     * @returns {number} Returns the remainder.
+     * @example
+     *
+     * _.modulus(6, 4);
+     * // => 2
+     */
+    function modulus(dividend, divisor) {
+      return dividend % divisor;
+    }
+
+    /**
+     * Calculates the result of a number raised to the power of another.
+     *
+     * @static
+     * @memberOf _
+     * @category Math
+     * @param {number} base
+     * @param {number} exponent
+     * @returns {number}
+     * @example
+     *
+     * _.power(6, 4);
+     * // => 1296
+     */
+    function power(base, exponent) {
+      return Math.pow(base, exponent);
+    }
+
+    /**
+     * Calculates the result of a number rounded to an optional precision.
+     *
+     * @static
+     * @memberOf _
+     * @category Math
+     * @param {number} number The number to round.
+     * @param {number} [precision] The precision to round to.
+     * @returns {number} Returns the rounded number.
+     * @example
+     *
+     * _.round(4.006);
+     * // => 4
+     *
+     * _.round(4.006, 2);
+     * // => 4.01
+     *
+     * _.round(4060, -2);
+     * // => 4100
+     */
+    function round(number, precision) {
+      if (precision) {
+        precision = power(10, precision);
+        return Math.round(number * precision) / precision;
+      } else {
+        return Math.round(number);
+      }
+    }
+
+    /**
+     * Calculates the sum of an array of numbers.
+     *
+     * @static
+     * @memberOf _
+     * @category Math
+     * @param {number} array The numbers to add.
+     * @returns {number} Returns the sum.
+     * @example
+     *
+     * _.sum([4, 6, 2]);
+     * // => 12
+     */
+    function sum(array) {
+      return reduce(array, add);
+    }
+
+    /**
+     * Calculates the mean of an array of numbers.
+     *
+     * @static
+     * @memberOf _
+     * @category Math
+     * @param {number} array The numbers to average.
+     * @returns {number} Returns the mean.
+     * @example
+     *
+     * _.mean([4, 6, 2]);
+     * // => 4
+     *
+     * _.mean([4, 6, 2, 2]);
+     * // => 3.5
+     */
+    function mean(array) {
+      if (!isArray(array)) return;
+      return sum(array) / array.length;
+    }
+
+    /**
+     * Calculates the median of an array of numbers.
+     *
+     * @static
+     * @memberOf _
+     * @category Math
+     * @param {number} array The numbers to caculate the median of.
+     * @returns {number} Returns the median.
+     * @example
+     *
+     * _.median([4, 6, 2]);
+     * // => 4
+     *
+     * _.median([4, 6, 2, 2]);
+     * // => 3
+     */
+    function median(array) {
+      if (!isArray(array)) return;
```

Comment:
Would that include `_.median(1);`? Also, is it worth supporting `_.median(1, 2, 3);`?